shared/src/ledger/eth_bridge/bridge_pool_vp.rs

```diff
@@ -116,11 +118,29 @@ where
         };
 
         let pending_key = get_pending_key(&transfer);
+        // check that transfer is not already in the pool
+        match (&self.ctx).read_pre(&pending_key) {
+            Ok(Some(_)) => {
+                tracing::debug!(
+                    "Rejecting transaction as the transfer is already in the \
+                     Ethereum bridge pool."
+                );
+                return Ok(false);
+            }
+            Err(_) => {
```

Comment:
can the returned error wrap this error, or can we log it if we will not propagate it (at least at debug?)